src/components/account-deletions-report.tsx

```diff
@@ -0,0 +1,84 @@
+import { useContext, useRef } from "preact/hooks";
+import { ascending, descending, flatGroup } from "d3-array";
+import * as Plot from "@observablehq/plot";
+import useRegistrationData from "../hooks/use-registration-data";
+import useElementWidth from "../hooks/use-element-width";
+import { ReportFilterContext } from "../contexts/report-filter-context";
+import { formatAsPercent, yearMonthDayFormat } from "../formats";
+import PlotComponent from "./plot";
+import Table from "./table";
+import type { ProcessedResult } from "../models/daily-registrations-report-data";
+import type { TableData } from "./table";
+import { format } from "d3-format";
+
+interface ProcessedFormattedData {
+  date: Date;
+  value: number;
+}
+
+function plot({ data, width }: { data: ProcessedFormattedData[]; width?: number }): HTMLElement {
+  return Plot.plot({
+    color: {
+      type: "ordinal",
+      scheme: "Tableau10",
+    },
+    marks: [Plot.ruleY([0]), Plot.line(data, { x: "date", y: "value" })],
+    width,
+    y: {
+      domain: [0, 0.1],
+    },
+  });
+}
+
+function tabulate(results: ProcessedFormattedData[]): TableData {
+  return {
+    header: ["Date", "Percent"],
+    body: results
+      .sort(({ date: aDate }, { date: bDate }) => ascending(aDate, bDate))
+      .map(({ date, value }) => [yearMonthDayFormat(date), value]),
+  };
+}
```

Comment:
the `tabulate` functions lend really well to unit testing, I'd recommend adding tests for this one if possible

Reply:
Yep, will plan to add specs 👍

Reply:
> the `tabulate` functions lend really well to unit testing, I'd recommend adding tests for this one if possible

Added tests in 19dd7e5.